src/app/utils/index.tsx

```diff
@@ -2,7 +2,6 @@ import { Auth } from "aws-amplify";
 
 async function getJwtToken() : Promise<string | void>{
   return Auth.currentSession().then((data) => {
-    console.log(data.getIdToken().getJwtToken());
```

Comment:
can we leave this here first? i need this to manually put in my backend api

Reply:
yup sure.